src/NuGet.Clients/NuGet.PackageManagement.UI/Models/ItemsChangeObservableCollectionBase.cs

```diff
@@ -13,9 +14,28 @@ namespace NuGet.PackageManagement.UI
     /// <typeparam name="T"></typeparam>
     public class ItemsChangeObservableCollection<T> : ObservableCollection<T>
     {
+        public ItemsChangeObservableCollection() : base()
```

Comment:
Could you move these everything after `:` to a new line? I forgot before that's in our coding guidelines.